environs/interface.go

```diff
@@ -339,16 +339,24 @@ type BootstrapEnviron interface {
 	// StorageProviders returned from Environ.StorageProvider will
 	// be scoped specifically to that Environ.
 	storage.ProviderRegistry
+}
+
+// ModelResources provides the API to use when creating new models.
```

Comment:
I think this could be better. In my opinion something like `// ModelResources provides the API for checking and instantiating models and their resources in a provider`

Reply:
k